### ISSUE TITLE
Custom Study deck with buried cards

### ISSUE THREAD

**Issue by tkrajina** [ankidroid/Anki-Android#5735]
###### Reproduction Steps

1. I reviewed all cards for that day, but some decks contain "buried" cards
2. Create a "Custom study" deck with cards I failed that day. That deck contain also some of the buried cards.
3. When I empty the "Custom Study" cards, all the cards are moved back to the original deck, but the buried cards aren't buried.

###### Expected Result

The buried cards should be returned in the original deck and still being "buried". But in reality it isn't.

###### Actual Result

With the "Custom Study" deck:

![Screenshot_20200203-174320](https://user-images.githubusercontent.com/379243/73672759-bcdb5a00-46ad-11ea-91a8-bb7dbe712a13.png)

After emptying the "Custom Study" deck:

![Screenshot_20200203-174335](https://user-images.githubusercontent.com/379243/73672761-bcdb5a00-46ad-11ea-9138-767c186935df.png)

I tried the same in the desktop Anki app,

With the "Custom Study" deck:

<img width="461" alt="Screenshot 2020-02-03 at 17 49 33" src="https://user-images.githubusercontent.com/379243/73672862-f2804300-46ad-11ea-94f2-c71a0cd4a487.png">

After emptying the "Custom Study" deck:

<img width="461" alt="Screenshot 2020-02-03 at 17 49 46" src="https://user-images.githubusercontent.com/379243/73672873-f7dd8d80-46ad-11ea-93e6-186181d6336e.png">

It seems the difference is that on Desktop Anki the "Custom Study" deck ignores the 4 buried cards (it contains 12 cards, vs 16 in AnkiDroid). When I empty it -- all 12 are returned to the original Deck. On AnkiDroid the 4 buried cards are returned but are left unburied in the original deck.  

The criteria for the "Custom Study" deck are `deck:"Langs::Slovenski" rated:1:1`. That is, all the cards I failed that day in the `Slovenski` deck. 

###### Debug info

I tried this in `2.9.1`:

```
AnkiDroid Version = 2.9.1
Android Version = 9
ACRA UUID = 1724f33f-bb39-4b76-bc63-a17f4b3bfea8
```

And `2.9.2`:

```
AnkiDroid Version = 2.9.2
Android Version = 9
ACRA UUID = 32eca7e6-6055-4270-86d0-1e9c54e5bd58
```

###### Research
*Enter an [ x ] character to confirm the points below:*

[x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[x] I have searched for similar existing issues here and on the user forum

**Comment by mikehardy** [comment]
Hi there! Are using scheduler v1 or v2?

**Comment by tkrajina** [comment]
@mikehardy sorry, forgot to mention that. I use v2.

**Comment by mikehardy** [comment]
Hey @hssm - is something that the v2 scheduler code would handle differently than v1? Looks like the root cause is that creation of custom study decks isn't ignoring buried cards, based on the good testing from @tkrajina above

**Comment by dae** [comment]
There's a bug in the computer version going in the opposite direction, so you may want to address that at the same time: https://github.com/ankitects/anki/commit/fe493e31c4d73ae2bbd0c4d8c6b835974c0e290c

**Comment by tkrajina** [comment]
Also, something to add here. Those 4 cards that are restored unburied -- were cards that I failed that day. So, I'd expect they really should have been part of the "Custom Study" deck. Like it really is in AnkiDroid (16 cards( but isn't in Anki Desktop (12 cards). I'm not sure if that ankitects/anki@fe493e3 fixes this problem or something else. 

Ideally, I think the custom study here should have 16 cards, but emptying it should restore even buried cards to still be buried. 

But, even a solution where custom study decks just ignore buried cards is better than the current situation - where I have to go through those 4 cards and bury them manually *again*. 

Anyway, this is how I would expect it to behave. It's now up to you to decide :)

**Comment by mikehardy** [comment]
This one is important and needs triage

**Comment by tkrajina** [comment]
I tried to reproduce this issue with the latest version, and seems it's fixed. 

Closing it now. Thank you very much!

**Comment by david-allison-1** [comment]
Thanks! 

Likely fixed in 2d00fff26e2f2fc487d91019a686454d16029c7c (#6408)
